src/python/pants/core/goals/test.py

```diff
@@ -147,9 +147,9 @@ def level(self) -> LogLevel:
 
     def message(self) -> str:
         if self.skipped:
-            return f"{self.address} skipped."
+            return "skipped."
         status = "succeeded" if self.exit_code == 0 else f"failed (exit code {self.exit_code})"
```

Comment:
If we include the ending `"."` in the status message here, we save ourselves from the format string when assigning `message` on line 152, or rather, why not assign directly to message here on this line, as status was only used to avoid duplicating the format string with `self.address` in it, fwict.